package.json

```diff
@@ -81,7 +81,7 @@
     "@testing-library/react": "11.1.2",
     "@testing-library/user-event": "10.4.1",
     "@woocommerce/components": "5.1.2",
-    "@woocommerce/currency": "2.0.0",
+    "@woocommerce/currency": "^3.0.0",
```

Comment:
I'd suggest using "3.0.0" here, so deps bumping remaining under our control (according to how we setup renovate bot).

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/1236/commits/679b298a29d2b7471026557d35da795397b5b257